fetch.bs

```diff
@@ -4203,6 +4208,9 @@ steps. They return a <a for=/>response</a>.
 
    <li><p><a>Append a request `<code>Origin</code>` header</a> for <var>httpRequest</var>.
 
+   <li><p><a abstract-op lt="set the Fetch metadata headers for a request">Set the Fetch metadata
+   headers for <var>request</var></a>. [[!FETCH-METADATA]]
```

Comment:
_httpRequest_, right?

Reply:
And ideally we would use "append" here as we do elsewhere since there should be nothing to override.

Reply:
Used the right variable, and renamed the algorithm. I'll update Fetch Metadata accordingly once you're happy with the rest.